vanilla-framework/vanilla-framework
# Update package.json deps

## Done

Updated gulp-sourcemaps and gulp-sass-lint deps

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/
- Ensure no errors when running `./run build`
